docsbox/docs/tasks.py

```diff
@@ -1,40 +1,43 @@
 import os
 import traceback
+import json
 
 from subprocess import run
 from wand.image import Image
 from img2pdf import convert as images_to_pdf
 from tempfile import NamedTemporaryFile, TemporaryDirectory
 from rq import get_current_job
-from docsbox import app, rq
-from docsbox.docs.utils import make_zip_archive, make_thumbnails, remove_xmp_meta, check_file_content, has_PDFA_XMP, remove_alpha, correct_orientation
-from docsbox.docs.via_controller import save_file_on_via
+from docsbox import app, rq, db
 from docsbox.docs.unoconv import UnoConverter
+from docsbox.docs.utils import *
+from docsbox.docs.via_controller import *
 
 
 def get_task(task_id):
     queue = rq.get_queue()
     return queue.fetch_job(task_id)
 
-
 def remove_file(path):
     """
     Just removes a file.
     Used for deleting original files (uploaded by user) and result files (result of converting)
     """
     return os.remove(path)
 
-
-def store_file(data, filename, stream=False):
-    suffix = os.path.splitext(filename)[1] if filename else ""
-    with NamedTemporaryFile(delete=False, dir=app.config["MEDIA_PATH"], suffix=suffix) as tmp_file:
-        if stream:
-            for chunk in data.iter_content(chunk_size=128):
-                tmp_file.write(chunk)
+@rq.job(timeout=app.config["REDIS_JOB_TIMEOUT"])
+def process_convertion_by_id(file_id, options, meta):
+    file_info = json.loads(db.get('fileId:' + file_id))
```

Comment:
If db.get() can return None, I think these calls will throw errors currently if that's the case?

Reply:
the entry should exist before task is queued but it still better to assume that it may cause error